Content.Client/VendingMachines/UI/VendingMachineMenu.xaml.cs

```diff
@@ -21,16 +21,46 @@ public sealed partial class VendingMachineMenu : FancyWindow
         [Dependency] private readonly IEntityManager _entityManager = default!;
 
         private readonly Dictionary<EntProtoId, EntityUid> _dummies = [];
-        private readonly Dictionary<EntProtoId, (ListContainerButton Button, VendingMachineItem Item)> _listItems = new();
-        private readonly Dictionary<EntProtoId, uint> _amounts = new();
+    private readonly Dictionary<EntProtoId, (ListContainerButton Button, VendingMachineItem Item)> _listItems = new(); // 🌟Starlight🌟
+    private readonly Dictionary<EntProtoId, uint> _amounts = new(); // 🌟Starlight🌟
+    private readonly Dictionary<EntProtoId, int> _prices = new(); // 🌟Starlight🌟
 
         /// <summary>
         /// Whether the vending machine is able to be interacted with or not.
         /// </summary>
-        private bool _enabled;
+    private bool _enabled; // 🌟Starlight🌟
+    private bool _showPrices; // 🌟Starlight🌟
 
         public event Action<GUIBoundKeyEventArgs, ListData>? OnItemSelected;
 
+        // 🌟Starlight🌟 start
+
+        /// <summary>
+        /// Updates the balance display
+        /// </summary>
+        /// <param name="balance">Current player balance</param>
+        public void UpdateBalance(int balance)
+        {
+            BalanceLabel.Text = $"Balance: {balance}₡";
+        }
```

Comment:
```suggestion
        public void UpdateBalance(int balance) => BalanceLabel.Text = $"Balance: {balance}₡";
```

Make this as lambda, why? It's not about code style(but maybe, idk), it's just for more compactness.